dev/maven-plugin/src/main/scala/com/lightbend/lagom/maven/ServerMojos.scala

```diff
@@ -152,12 +158,10 @@ class StartServiceLocatorMojo @Inject() (logger: MavenLoggerProxy, facade: Maven
 
       val scalaClassLoader = scalaClassLoaderManager.extractScalaClassLoader(cp)
 
-      val scalaUnmanagedServices =
-        if (cassandraEnabled) {
-          StaticServiceLocations.withCassandraLocation(cassandraPort, unmanagedServices.asScala.toMap)
-        } else {
-          unmanagedServices.asScala.toMap
-        }
+      val theKafkaAddress = if (kafkaAddress == null) s"127.0.0.1:$kafkaPort" else kafkaAddress
+
+      val scalaUnmanagedServices = StaticServiceLocations.staticServiceLocations(cassandraPort, theKafkaAddress) ++
```

Comment:
Also, if `kafkaEnabled == true` and user provides a `kafkaAddress` we should raise an error since those seems exclusive setups.